install.json

```diff
@@ -1,54 +1,101 @@
 {
     "resources": {
-        "body": [
+        "head": [
+            {
+                "type": "style",
+                "src": "progress-bar.css"
+            },
             {
                 "type": "script",
-                "src": "script.js"
+                "src": "progress-bar.js"
+            }
+        ]
+    },
+    "preview": {
+        "handlers": [
+            {
+                "options": ["*"],
+                "execute": "ScrollProgressBar.setOptions(INSTALL_OPTIONS)"
             }
         ]
     },
     "options": {
         "properties": {
-            "Position": {
+            "position": {
                 "order": 1,
-                "title": "Height in pixels",
+                "title": "Bar Location",
                 "type": "string",
+                "format": "radios",
                 "enum": [
                     "top",
                     "bottom"
                 ],
                 "enumNames": {
-                    "top": "top",
-                    "bottom": "bottom"
+                    "top": "Top of the screen",
+                    "bottom": "Bottom of the screen"
                 },
                 "default": "top"
             },
-            "Height": {
+            "thickness": {
```

Comment:
Specifying the thickness as words, rather than a pixel number, makes the app more friendly to people who may not be familiar with pixels.